charts/base/values.yaml

```diff
@@ -58,9 +58,28 @@ ingress:
             servicePort: 80
           path: /*
   tls: []
-  #  - secretName: chart-example-tls
-  #    hosts:
-  #      - chart-example.local
+
+# secretsDefaultEngine: "ExternalSecrets"
```

Comment:
seems this field should not be commented, and its value should be used as intended, so that the case if "ExternalSecrets" should be handled instead of just checking if the value is set or not if above templates,

also why you removed/commented "secrets" field? I think you will got helm error when one do not use secrets so that no "secrets" field in their values.yaml, please check this case also